src/pepr/uds-operator-config/values.yaml

```
@@ -2,18 +2,35 @@
 # SPDX-License-Identifier: AGPL-3.0-or-later OR LicenseRef-Defense-Unicorns-Commercial
 
 operator:
+  ### DEPRECATED: This section is deprecated and will be removed in a future release. ###
   # Domain configuration (admin defaults to `admin.UDS_DOMAIN`)
   UDS_DOMAIN: "###ZARF_VAR_DOMAIN###"
   UDS_ADMIN_DOMAIN: "###ZARF_VAR_ADMIN_DOMAIN###"
   UDS_CA_CERT: "###ZARF_VAR_CA_CERT###"
   UDS_ALLOW_ALL_NS_EXEMPTIONS: "###ZARF_VAR_ALLOW_ALL_NS_EXEMPTIONS###"
   UDS_LOG_LEVEL: "###ZARF_VAR_UDS_LOG_LEVEL###"
-  AUTHSERVICE_REDIS_URI: "###ZARF_VAR_AUTHSERVICE_REDIS_URI###"
   KUBEAPI_CIDR: ""
   KUBENODE_CIDRS: ""
+  ### END DEPRECATED ###
+  AUTHSERVICE_REDIS_URI: "###ZARF_VAR_AUTHSERVICE_REDIS_URI###"
   # Allow Pepr watch to be configurable to react to dropped connections faster
   PEPR_LAST_SEEN_LIMIT_SECONDS: "300"
   # Allow Pepr to re-list resources more frequently to avoid missing resources
   PEPR_RELIST_INTERVAL_SECONDS: "600"
   # Configure Pepr reconcile strategy to have separate queues for faster reconciliation
   PEPR_RECONCILE_STRATEGY: "kindNsName"
+
+cluster:
+  attributes:
+    UDS_CLUSTER_NAME: "###ZARF_VAR_CLUSTER_NAME###"
+    UDS_CLUSTER_TAGS: "###ZARF_VAR_CLUSTER_TAGS###"
+  expose:
+    # Domain configuration (admin defaults to `admin.UDS_DOMAIN`)
+    UDS_DOMAIN: "###ZARF_VAR_DOMAIN###"
+    UDS_ADMIN_DOMAIN: "###ZARF_VAR_ADMIN_DOMAIN###"
+    UDS_CA_CERT: "###ZARF_VAR_CA_CERT###"
+  policy:
+    UDS_ALLOW_ALL_NS_EXEMPTIONS: "###ZARF_VAR_ALLOW_ALL_NS_EXEMPTIONS###"
+  networking:
+    KUBEAPI_CIDR: ""
+    KUBENODE_CIDRS: []
```

imo all of these can follow better values naming standards. As long as we are maintaining parity on the zarf vars, these are net new keys so we can name them better. For example:
```suggestion
cluster:
  attributes:
    clusterName: "###ZARF_VAR_CLUSTER_NAME###"
    clusterTags: "###ZARF_VAR_CLUSTER_TAGS###"
  expose:
    # Domain configuration (admin defaults to `admin.<domain>`)
    domain: "###ZARF_VAR_DOMAIN###"
    adminDomain: "###ZARF_VAR_ADMIN_DOMAIN###"
    caCert: "###ZARF_VAR_CA_CERT###"
  policy:
    allowAllNsExemptions: "###ZARF_VAR_ALLOW_ALL_NS_EXEMPTIONS###"
  networking:
    kubeApiCidrs: ""
    kubeNodeCidrs: []
```

For context the other keys were all caps + `_` to match with env standards since they were directly mounted as env by pepr. Since we're no longer doing that it would be good to align with helm norms of camelcase here.